Examples/Sources/ChatbotExample/Views/MainChatView.swift

```diff
@@ -0,0 +1,156 @@
+                    errorMessage: viewModel.errorMessageBinding,
+                    isLoading: viewModel.isLoading,
+                    messageCount: viewModel.currentThreadMessages.count,
+                    onSend: viewModel.sendMessage
+                )
+            }
+        }
+    }
+}
+
+// MARK: - Chat Header View
+
+struct ChatHeaderView: View {
+    var viewModel: ChatbotViewModel
+    
+    var body: some View {
+        HStack {
+            // Thread title or placeholder
+            VStack(alignment: .leading, spacing: AppSpacing.xs) {
+                if let thread = viewModel.selectedThread {
+                    Text(thread.title)
+                        .font(AppFonts.title2)
+                        .fontWeight(.semibold)
+                        .foregroundColor(AppColors.text)
+                    Text("Created today")
+                        .font(AppFonts.caption)
+                        .foregroundColor(AppColors.textSecondary)
+                } else {
+                    Text("AI Chat Assistant")
+                        .font(AppFonts.title2)
+                        .fontWeight(.semibold)
+                        .foregroundColor(AppColors.text)
+                }
+            }
+            
+            Spacer()
+            
+            Button("⚙️") {
+                viewModel.toggleSettings()
+            }
+            .iconButtonLargeStyle()
+        }
+        .padding(.horizontal, AppSpacing.xl)
+        .padding(.vertical, AppSpacing.lg)
+        .background(AppColors.background)
+    }
+}
+
+// MARK: - Chat Content View
+
+struct ChatContentView: View {
+    var viewModel: ChatbotViewModel
+    
+    var body: some View {
+        ChatMessagesView(
+            messages: viewModel.currentThreadMessages,
+            isLoading: viewModel.isLoading
+        )
+    }
+}
+
+// MARK: - Empty Thread View
+
+struct EmptyThreadView: View {
+    var viewModel: ChatbotViewModel
+    
+    var body: some View {
+        VStack {
+            Spacer()
+            
+            VStack(spacing: AppSpacing.xl) {
+                VStack(spacing: AppSpacing.lg) {
+                    Text("🤖")
+                        .font(.system(size: 64))
+                    
+                    VStack(spacing: AppSpacing.md) {
+                        Text("Ready to Chat")
+                            .font(AppFonts.title)
+                            .fontWeight(.bold)
+                            .foregroundColor(AppColors.text)
+                        
+                        Text("Create a new conversation to start chatting with AI.\nYour conversations will be saved and you can continue them anytime.")
+                            .font(AppFonts.body)
+                            .foregroundColor(AppColors.textSecondary)
+                            .multilineTextAlignment(.center)
+                    }
+                }
+                
+                Button("+ Start New Conversation") {
```

Comment:
Change title to `Start new conversion` to match general SwiftCrossUI button naming convention among the other examples.